src/content/developers/docs/nodes-and-clients/archive-nodes/index.md

```diff
@@ -0,0 +1,81 @@
+
+As explained above, a full node would need to generate this data by EVM execution which uses the CPU and takes time. Archive nodes access them on the disk and serves responses immediately. This is useful feature for certain parts of infrastracture, for example: 
+
+* Service providers like block explorers
+* Researchers
+* Security analysts
+* Dapp developers
+* Auditing and compliance 
+
+There are various free [services](/developers/docs/nodes-and-clients/nodes-as-a-service/) that also allow access to historical data. As it is more demanding to run an archive node, this access is mostly limited and works only for occasional access. If your project requires constant access to historical data, you should consider running one yourself.  
+
+## Implementations and usage
+
+Archive node in this context means data served by user facing EL clients as they handle the state database and provide JSON-RPC endpoint. Configuration options, sync time and database size may vary by client. For details, please refer to the documentation provided by your client. 
+
+Before starting your own archive node, learn about the differences between the clients and especially the various [hardware requirements](/developers/docs/nodes-and-clients/run-a-node/#requirements). Most clients are not optimized for this feature and their archives requirs more than 12TB of space. In contrast, implementations like Erigon and Akula can store the same data in under 3TB which makes them the most effective way of running an archive node.
+
+## Recommended practices
+
+Apart from general [recommendations for running](developers/docs/nodes-and-clients/run-a-node/), an archive node may be more demanding on hardware and maintanance. Considering its [key features](https://github.com/ledgerwatch/erigon#key-features), the most practical approach is using[Erigon](https://ethereum.org/en/developers/docs/nodes-and-clients/#erigon) client implementation.
+
+### Hardware 
+
+Always make sure to verify hardware requirements in given mode in client's documentation. 
+et
+The biggest requirement which comes with archive node is the disk space. Depending on client, it varies from 3 to 12TB. Even if HDD might be considered a better solution for large amounts of data, syncing it and constantly updating the head of the chain will require SSD drives. SATA drives are good enough but it should be a reliable quality, at least TLC. Disks can be fitted into a desktop computer or a server with enough slots. Such dedicated devices are ideal for running high uptime node. It's totally possible to run it on a laptop but the portability will come at an aditional cost.  
+
+All of the data needs to be fit in one volume, therfore disks has to be joint, e.g. with RAID0 or LVM. It might be also worth considering using [ZFS](https://en.wikipedia.org/wiki/ZFS) as it supports "Copy-on-write" which ensures data is correctly written to the disk without any low level errors. 
```

Comment:
```suggestion
All of the data needs to be fit in one volume, therfore disks have to be joined, e.g. with [RAID0](https://en.wikipedia.org/wiki/Standard_RAID_levels#RAID_0) or [LVM](https://web.mit.edu/rhel-doc/5/RHEL-5-manual/Deployment_Guide-en-US/ch-lvm.html). It might be also worth considering using [ZFS](https://en.wikipedia.org/wiki/ZFS) as it supports "Copy-on-write" which ensures data is correctly written to the disk without any low level errors. 
```